gluten-ut/spark34/src/test/scala/org/apache/gluten/utils/velox/VeloxTestSettings.scala

```diff
@@ -144,6 +144,7 @@ class VeloxTestSettings extends BackendTestSettings {
   enableSuite[GlutenRegexpExpressionsSuite]
   enableSuite[GlutenSortOrderExpressionsSuite]
   enableSuite[GlutenStringExpressionsSuite]
+    .exclude("base64/unbase64 for string")
```

Comment:
Could you clarify why we need to remove this test? Thanks!

Reply:
> Could you clarify why we need to remove this test? Thanks!

@sujithjay @PHILO-HE There seems some differences between the unbase64 function provided by Velox and Spark, such as when the length of the passed string is not a multiple of 4.

Reply:
@sujithjay, it would be better to firstly investigate the differences. If only corner cases are involved, we can ignore the difference. Otherwise, better to implement a new Velox function for Spark.

Reply:
I will remove the unnecessary test exclusion.

Reply:
@sujithjay It's perfect, Could you do a test? It seems if it's not a multiple of 4 then the spark will return null, but the velox will crash (throw an exception) . The reason for adding this filter is because of this difference. Thanks.

Reply:
@fyp711 , @sujithjay , @PHILO-HE, note that there are a few failure mode behavior differences between velox and spark, all boiling down to the java implementation of base64 decoding, so it is a bit more than just returning null vs. exception on non-multiples of 4. Some examples in Spark, gleaned from looking at Java sources:

* Extraneous padding is ignored. For example, "YQ=====" decodes to "a"
* Missing padding is ignored in some cases. For example, "aA" is decoded to "h"
* Invalid characters in some cases are ignored. For example, "@@dmVsb3g=" decodes to "velox".

Reply:
@supermem613, thanks a lot for your information! @sujithjay, could you help confirm? We may need to align the implementation with Spark.

Reply:
Thank you @supermem613. If you could share any tests you used to compare these cases, that would be helpful.

Reply:
Sure. Here are some:

"YQ=" -> "a"
"YQ===" -> "a"
"" -> ""

"aA" " -> "h"
"c3d" -> "sw"
"cd@" -> "q"

"SGVsbG8gV29ybGQgZnJvbSBWZW@@xveCE=" -> "Hello World from Velox!"
"@@dmVsb3g=" => "velox"

"a" -> exception
"c@" -> exception
"==" -> exception
"SGVsbG8gV29ybGQgZnJvbSBWZW===xveCE=" -> exception

Reply:
@PHILO-HE I confirm that some of the above cases shared by @supermem613 show behavior differences for Spark and Velox. Do you agree that we should implement `base64` and `unbase64` in Velox that behaves the same as Spark? If yes, I will close this PR and start implementing them on Velox.

Reply:
@sujithjay, yes, it would be nice to add a new implementation in Velox. Thanks!